org.civicrm.bootstrapcivicrm/scss/bootstrap/overrides/style/civihr-ui-select/_result-list.scss

```diff
@@ -0,0 +1,16 @@
+.result-list {
+  &__contact-item {
+    clear: left;
+
+    &::before {
+      content: '\f007';
```

Comment:
I prefer to use it [this refactoring](https://github.com/civicrm/civihr/pull/1109/files#diff-f701cc19e9fc22db8a3443c6ff8c9503L1) is merged